src/InstallTool.php

```diff
@@ -121,11 +121,11 @@ public function setConnection(Connection $connection): void
     /**
      * Checks if a database connection can be established.
      *
-     * @param string $name
+     * @param string|null $name
      *
      * @return bool
      */
-    public function canConnectToDatabase(string $name): bool
+    public function canConnectToDatabase(string $name = null): bool
```

Comment:
This is wrong, because it says "there can be a name but it is optional". However, our case is "there must be a name but it can be null", which translates to:

```php
public function canConnectToDatabase(?string $name): bool
```